package.json

```diff
@@ -46,7 +41,12 @@
     "test": "jest",
     "test:watch": "npm run test -- --watch",
     "test:coverage": "npm run test -- --coverage",
-    "open:coverage": "npm run test:coverage && opn ./coverage/lcov-report/index.html"
+    "open:coverage":
+      "npm run test:coverage && opn ./coverage/lcov-report/index.html",
+    "build:server": "rimraf ./dist &&  babel api --out-dir ./dist/api",
+    "build:client": "webpack -p --config webpack.prod.config.js --progress",
```

Comment:
@jbuck94 thanks for create this PR! I tried the branch on my side. There is a missing package here and is "html-webpack-plugin"